test/chrome_test.dart

```diff
@@ -73,18 +73,21 @@ void main() {
 
     for (var signIn in [false, true]) {
       group('and signIn = $signIn', () {
-        setUpAll(() {
+        setUp(() {
           dataDir = Directory.systemTemp.createTempSync(_userDataDirName);
         });
 
-        tearDownAll(() {
-          var deleted = false;
+        tearDown(() async {
```

Comment:
The test infra has retry logic, too...

Reply:
See https://pub.dev/documentation/test_core/latest/test_core.scaffolding/test.html

Reply:
I couldn't find any retry logic for `tearDown` only, am I missing something? I think chrome needs some time to close before its data directory is actually removable, hence the wait and retries...

Reply:
I think I found another potential source of flakiness. Will send a fix.

Reply:
k

Reply:
how does it look now?